.nvmrc

```diff
@@ -0,0 +1 @@
+v18.19.0
```

Comment:
Does this enforce a particular node version? I'm currently on 21.5.0

Reply:
only if you run ``nvm use``